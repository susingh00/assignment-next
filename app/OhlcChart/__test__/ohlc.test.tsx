import { render } from "@testing-library/react";
import { OhlcChart } from "..";
import renderer from "react-test-renderer";
describe("OHLC", () => {
  it("Render OHLC Screen", async () => {
    render(<OhlcChart />);
    const tree = renderer.create(<OhlcChart />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
