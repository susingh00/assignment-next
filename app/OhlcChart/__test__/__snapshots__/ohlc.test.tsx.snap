// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OHLC Render OHLC Screen 1`] = `null`;
